src/gameplay_v2/roles/mordred.ts

```diff
@@ -0,0 +1,12 @@
+import Entity from './entity';
+import { Alliance } from '../gameTypes';
+
+export default class Mordred extends Entity {
+  description: string;
+
+  constructor() {
+    super(Alliance.Resistance);
+    this.description = 'A spy who is invisible to Merlin.';
+    //this.components.push(new hideFromMerlinC());  
```

Comment:
Shouldn't need this component. If we have a SeeSpies component and a HideFromMerlin component, getting the two to interact correctly with each other can be a nightmare. Let's just have this logic in SeeSpies() to filter out mordred if the user is merlin. Maybe we'll need to add this data to the `data` field in the SeeSpiesC.

Reply:
Removed HidefromMerlin component :)